### PR TITLE
added ticket 6 list of comments

### PR DESCRIPTION
**Pull request by Anna702**

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+  useEffect(() => {
+    console.log(article_id);
+    setIsLoading(true);
+    getComments(article_id).then((comments) => {
+      setComments(comments);
+      setIsLoading(false);
+    });
+  }, [article_id]);
+
+  return (
```
Comment (JVal-NC) [inline src/components/Comments.jsx:17]: Could you include something to indicate if there are no comments for the article?

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+      ) : (
+        <ul className="comments_list">
+          {comments.map((comment) => {
+            return (
+              <li className="comment_one" key={comment.comment_id}>
+                <p id="comment_body">{comment.body}</p>
+                <h4 id="comment_author">Created by: {comment.author}</h4>
+                <h5 id="comment_added_at">
+                  Added:
+                  {comment.created_at}
```
Comment (JVal-NC) [inline src/components/Comments.jsx]: Could the date be formatted to be show a more typical date format?

File: src/components/ArticleCard.jsx [hunk 3]
```diff
@@ -23,25 +23,28 @@ const ArticleCard = () => {
...
-            src={article.article_img_url}
-          />
-          <p id="card_article_body">{article.body}</p>
-
+        <div>
+          <div className="card_article">
+            <h2 id="card_article_title">{article.title}</h2>
+            <h4 id="card_article_card_topic">
+              Topic: &nbsp;
+              <Link to={"/articles?topic=" + article.topic}>
```
Comment (JVal-NC) [inline src/components/ArticleCard.jsx]: This appears to link to a url that doesn't currently work? Suspect this is for a future ticket. Please remove for now.

However you do need a way to get back to the home page from the individual review page, please include a way to do this.

**Review by JVal-NC — COMMENTED** [review]
Hi Anna, few comments to consider but good work so far.

Please do another PR once comments have been addressed.

**Review by pippaamy — APPROVED** [review]
Good work Anna, go ahead and merge!

**Pull request merged**